cloudigrade/internal/authentication.py

```diff
@@ -35,3 +35,19 @@ class IdentityHeaderAuthenticationInternalCreateUser(IdentityHeaderAuthenticatio
     require_account_number = True
     require_user = False
     create_user = True
+
+
+class IdentityHeaderAuthenticationInternalConcurrent(IdentityHeaderAuthentication):
```

Comment:
I like this idea so much that I think we could use it in more internal APIs! Maybe not right now but in the near future. So, rather than tie this name specifically to the concurrent API, how about something like this?

```suggestion
class IdentityHeaderAuthenticationInternalFakeIdentity(IdentityHeaderAuthentication):
```

Maybe that's not the best name either… but something along those lines?